src/awkward/operations/ak_linear_fit.py

```diff
@@ -185,10 +187,10 @@ def _impl(x, y, weight, axis, keepdims, mask_identity):
                 behavior=behavior,
             )
         delta = (sumw * sumwxx) - (sumwx * sumwx)
-        intercept = nplike.true_divide(((sumwxx * sumwy) - (sumwx * sumwxy)), delta)
-        slope = nplike.true_divide(((sumw * sumwxy) - (sumwx * sumwy)), delta)
-        intercept_error = nplike.sqrt(nplike.true_divide(sumwxx, delta))
-        slope_error = nplike.sqrt(nplike.true_divide(sumw, delta))
+        intercept = ((sumwxx * sumwy) - (sumwx * sumwxy)) / delta
+        slope = ((sumw * sumwxy) - (sumwx * sumwy)) / delta
+        intercept_error = _numpy_ufuncs.sqrt(sumwxx / delta)
+        slope_error = _numpy_ufuncs.sqrt(sumw / delta)
```

Comment:
I was originally thinking that we wanted to drop the use of `import numpy` within Awkward so that we can be confident no code is sneakily using it by accident.  However, the problem is that we need to invoke a ufunc via Awkward's ufunc machinery. Elsewhere, we also need the `np.ufunc` symbols in the modules that register awkward behaviours.

A forthcoming PR will replace the `NumpyMetadata` class with a `metadata` module, but that's for dtypes and constants rather than ufuncs. 

So, my solution here is to explicitly import numpy as `_numpy_ufuncs`, so that we could hypothetically implement a plugin to only allow imports of numpy that reflect it's intention to be used for e.g ufuncs only. It seems like a least-worst option.

Reply:
As long as that's how it's done consistently. The

```python
import numpy as _numpy_ufuncs   # be explicit about where NumPy is directly used
```

should probably have a comment everywhere it appears, so that someone else doesn't look at it, scratch their head, and change it back.